WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -62,6 +64,13 @@ class PostListItemViewHolder(
         }
     }
 
+    private fun updateStatusLabels(view: WPTextView, statusLabels: List<UiString>, delimiter: UiString) {
+        view.text = StringUtils.join(
```

Comment:
We don't need `StringUtils.join` anymore. Kotlin has [`joinToString`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html). 🙂 

```kotlin
        val separator = uiHelpers.getTextOfUiString(view.context, delimiter)
        view.text = statusLabels.joinToString(separator) { uiHelpers.getTextOfUiString(view.context, it) }
```

Reply:
👍 Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9491/commits/2189fb6bb61630095fc9145472740c00a5c99099